### ISSUE TITLE
AudioWorklet support

### ISSUE THREAD

**Issue by Bunagana** [brion/audio-feeder#37]
Hi @brion, hope all is well. I saw a comment in web-audio-backend about adding support for Audio Worker and I was wondering if you were still planning to add AudioWorklet support as the ScriptProcessorNode has been deprecated.

Thank you for your work on the audio-feeder.

**Comment by bvibber** [comment]
Yes in fact! :) I'm doing some modernization and internals cleanup all around and am definitely interested in figuring out if I can use audio worklets directly here; there's some potential big benefit for avoiding drops from a busy main thread if the data source can also run on a worker and send data directly in through a message port.

I'd have to make a new message port-based async API for that use case, since you can't just pass the object down to a worker to call methods on.

**Comment by bvibber** [comment]
(in progress)
